Tutorial.md

```diff
@@ -0,0 +1,178 @@
+# Welcome to the BiobakeryUtils tutorial
+
+## Using BiobakeryUtils with [MetaPhlAn](https://github.com/biobakery/MetaPhlAn/wiki/MetaPhlAn-3.0) files
+
+### Input file format
+
+BiobakeryUtils accepts `.tsv` files outputted from MetaPhlAn.
+Files can either contain a merged table or one or multiple single table(s).
+Example files that are used in this tutorial can be found in [BiobakeryUtils.jl/test/files/](https://github.com/BioJulia/BiobakeryUtils.jl/tree/main/test/files).
+
+### Creating CommunityProfiles
+
+A CommunityProfile can be created from MetaPhlAn file(s) using `metaphlan_profile` and `metaphlan_profiles`.
```

Comment:
This file should probably be in `docs/src/tutorial.md`, so that it will be built with Documenter.jl. Then here you can do things like 

```md
A [`CommunityProfile`](@ref) can be created from MetaPhlAn file(s) using [`metaphlan_profile`](@ref) and [`metaphlan_profiles`](@ref).
```

And those things will link to the appropriate docs.

Reply:
I added the [` `](@ref) but I'm not sure what it's supposed to do? The links lead to 404s

Reply:
At the top of the page, you might need to do

````
```@meta
CurrentModule = BiobakeryUtils
```
````
?

Don't worry about it too much - docs here need a bunch of work. See also #57  #48

Reply:
Okay, so should I revert it back to what it was before? What is the plan of action?